include/glow/LLVMIRCodeGen/GlowJIT.h

```diff
@@ -79,6 +79,9 @@ class GlowJIT {
       irStaticDestructorRunners_;
 #endif
 
+  llvm::DenseSet<llvm::orc::VModuleKey> vModKeys_;
```

Comment:
Could you move this data member definition to the rest of data member definitions?

Also, could add Doxygen comment to both new class members, i.e. to vModKeys_ and  resolveSymbol?

Reply:
Done.